module08/en.subject.tex

```diff
@@ -518,7 +518,7 @@ \section*{Examples}
 y_hat2 = logistic_predict(x2, theta2)
 vec_log_loss_(y2, y_hat2)
 # Output:
-2.4825011602474483
+2.4825011602472347
 
 # Example 3:
```

Comment:
Did you check for the example 3 ?
I got the following result with my code:
```python
res = 2.993853310859968
```

Reply:
you commented twice on the same example with different results on both. which results did you get ?

Reply:
No I comment here for the third example. The comment above is for the 2nd example.
I cannot comment the line corresponding to the value of the vec_loss_loss_ output.
So here is for the value on line 530

Reply:
@RobertsOwen